scripts/obtain/dotnet-install.ps1

```diff
@@ -182,7 +182,7 @@ function GetHTTPResponse([Uri] $Uri)
                 $HttpClientHandler = New-Object System.Net.Http.HttpClientHandler
                 $HttpClientHandler.Proxy =  New-Object System.Net.WebProxy -Property @{Address=$ProxyAddress;UseDefaultCredentials=$ProxyUseDefaultCredentials}
                 $HttpClient = New-Object System.Net.Http.HttpClient -ArgumentList $HttpClientHandler
-            } 
+            }
```

Comment:
nit: could you keep all of the whitespace changes out of the PR. It's distracting to review.